crates/ruma-events/tests/it/ui/08-enum-invalid-path.rs

```diff
@@ -1,4 +1,4 @@
-use ruma_common::events;
+use ruma_events as events;
```

Comment:
Hmmm, what is this about?

Reply:
Too lazy to change it and fix the error message. :stuck_out_tongue: